helm/values.yaml

```diff
@@ -122,4 +127,29 @@ route:
   timeout:
     enabled: false # defaults to 30s by openshift
     duration: 60s # supported units (us, ms, s, m, h, d)
-    
\ No newline at end of file
+    
+local:
+  s3:
+    secret:
+      create: false
+      accessKeyId: minikube
+      secretAccessKey: minikube
+  storage:
+    internalPv:
+      create: false #for local deployment only
+      name: local-tiles-pv
+      storage: 5Gi
+      hostPath: '/tiles' #tiles hostPatch
+    ingestionPv:
+      create: false #for local deployment only
+      name: local-sources-pv
+      storage: 5Gi
+      hostPath: /layerSources
+    internalPvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    ingestionSourcePvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    accessModes: ReadWriteOnce
```

Comment:
why once and not many?

Reply:
bug from the past i didnt touch it on this PR but fixed